### PR TITLE
Added more badges to the README

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/mxcube/HardwareRepository.svg?branch=master)](https://travis-ci.org/mxcube/HardwareRepository)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```
Comment (marcus-oscarsson) [inline README.md]: Not sure about this one, I agree that its nice with something the says that we are using a code style. However we are not really applying black on a regular basis

Comment (IvarsKarpics) [reply README.md]: Hmm, we did apply autopep and black to all HardwareRep. during the developers meeting. Why not to inform people that we use black coding guidelines?

Comment (marcus-oscarsson) [reply README.md]: We did, but then we should also have something that ensures that we are following them or at-least agree that we run black on a regular basis

Comment (IvarsKarpics) [reply README.md]: Ok, I removed the badge. During the next face to face meeting we can discuss how we can follow the black guidelines.

**Pull request merged**